### DISCUSSION TITLE
github student

### DISCUSSION THREAD

**Discussion by bohsa** [community/community#131025] [GitHub Education] [answered]
### Hi everyone,

i have a problem, i got my account accepted (after many tries) to github student, but its been a couple of days and i didnt recieve anything yet, any help?

**Comment by davevad93** [comment, marked as answer]
Hi @bohsa ,

It may take up to a couple of weeks for the benefits to become available, it usually takes just a few days but the waiting time may vary depending on different factors such as the volume of applications.

When the benefits are available the status label will turn purple and change to:

`Congrats! Your request for @bohsa was approved and your benefits are now available`

**Reply by bohsa** [reply]
thank you

**Reply by davevad93** [reply]
You're welcome! 👍
